src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs

```diff
@@ -246,8 +246,9 @@ public static OrderablePartitioner<TSource> Create<TSource>(IEnumerable<TSource>
             int coreOversubscriptionRate = 3;
```

Comment:
As long as you're changing this code, it'd be nice to make this a const and change the lowercase "Core" to "Core".

Reply:
replaced.